### ISSUE TITLE
Availability for R version 3.3.1

### ISSUE THREAD

**Issue by josefinacmenendez** [bd2kccd/r-causal#17]
Hello! Will this package be available for the latest R version in the future?

Thanks,

-Josefina

**Comment by chirayukong** [comment]
Absolutely. In fact, you can change the R version requirement in the DESCRIPTION file without a problem and re-create the r-causal package.
